teps/0091-trusted-resources.md

```diff
@@ -364,26 +372,161 @@ type KeylessRef struct {
+
+Need to create conditions for trusted resources so this can co-exist with current `apis.ConditionSucceeded` type:
+```go
+const (
+// ConditionTrustedResourcesVerified specifies that the resources pass trusted resources verification or not.
+ConditionTrustedResourcesVerified apis.ConditionType = "TrustedResourcesVerified"
+)
+```
+
+A successful condition is:
+ ```go
+ apis.Condition{
+			Type:    ConditionTrustedResourcesVerified,
+			Status:  corev1.ConditionTrue,
+			Reason:  podconvert.ReasonResourceVerificationSuccess
+			Message: "Trusted resource verification passed",
+		}
+ ```
+
+A failed condition is:
+  ```go
+ apis.Condition{
+			Type:    ConditionTrustedResourcesVerified,
+			Status:  corev1.ConditionFalse,
+			Reason:  podconvert.ReasonResourceVerificationFailed
+			Message: "", //filled with error message,
+		}
+ ```
+
+### How feature flag and Verification Policy update the status
+
+**No Matching Policies:**
+
+|                             | `Conditions.TrustedResourcesVerified` | `Conditions.Succeeded` |
+|-----------------------------|---------------------------------------|------------------------|
+| `no-match-policy`: "ignore" |                                       |                        |
+| `no-match-policy`: "warn"   | False                                 |                        |
+| `no-match-policy`: "fail"   | False                                 | False                  |
+
+**Matching Policies:**
+
+|                          | `Conditions.TrustedResourcesVerified` | `Conditions.Succeeded` |
+|--------------------------|---------------------------------------|------------------------|
+| all policies pass        | True                                  |                        |
+| any enforce policy fails | False                                 | False                  |
+| only warn policies fail  | False                                 |                        |
+
+**Examples:**
+* No Matching Policies:
+    * `trusted-resources-verification-no-match-policy` is set to `ignore`, then no updates on conditions, succeeded `conditions` are:
+      ```yaml
+      status:
+        conditions:
+        - lastTransitionTime: "2023-03-01T18:17:10Z"
+          message: All Steps have completed executing
+          reason: Succeeded
+          status: "True"
+          type: Succeeded
+      ```
+    * `trusted-resources-verification-no-match-policy` is set to `warn`, only add `false` `ConditionTrustedResourcesVerified` to `conditions`:
+      ```yaml
+      status:
+        conditions:
+        - lastTransitionTime: "2023-03-01T18:17:05Z"
+          message: Trusted resource verification failed
+          reason: ResourceVerificationFailed
+          status: "False"
+          type: TrustedResourcesVerified
+        - lastTransitionTime: "2023-03-01T18:17:10Z"
+          message: All Steps have completed executing
+          reason: Succeeded
+          status: "True"
+          type: Succeeded
+      ```
```

Comment:
in this case, we don't actually do anything to `Conditions.Succeeded` (there could be another failure, can't assume that it'd be "True")

Reply:
```suggestion
    * `trusted-resources-verification-no-match-policy` is set to `warn`, only add `false` `ConditionTrustedResourcesVerified` to `conditions`:
      ```yaml
      status:
        conditions:
        - lastTransitionTime: "2023-03-01T18:17:05Z"
          message: Trusted resource verification failed
          reason: ResourceVerificationFailed
          status: "False"
          type: TrustedResourcesVerified
      ```
```